### PR TITLE
Implement new subscription plans

### PR DESCRIPTION
**Pull request by benweissmann**
This PR:

- replaces the old DB-configured subscription terms with a new set of three plans: free, premium, and nonprofit.
- blocks users on the free plan from exporting their data or seeing voter-level results
- adds contact information to the subscription itself so we know who to contact about the subscription

### CODE REVIEW

File: app/integration/actionnetwork.py [hunk 1]
```diff
@@ -368,24 +368,22 @@ def sync_subscriber(subscriber: Client) -> None:
...
         "person": {
-            "given_name": interest.first_name,
-            "family_name": interest.last_name,
-            "email_addresses": [{"address": interest.email}],
+            "given_name": subscription.primary_contact_first_name,
+            "family_name": subscription.primary_contact_last_name,
+            "email_addresses": [{"address": subscription.primary_contact_email}],
             "custom_fields": {
-                "subscriber_is_c3": interest.nonprofit,
+                "subscriber_is_c3": subscription.plan == enums.SubscriberPlan.NONPROFIT,
```
Comment (benweissmann) [inline app/integration/actionnetwork.py:383]: @liewegas how can i test this? And do i need to pre-define these custom fields in AN?

Also, maybe we want to have this sync run nightly to pick up changes?

Comment (liewegas) [reply app/integration/actionnetwork.py:383]: There is a `Dev subscribers` actionnetwork group with an API key you can put in `.env` as `ACTIONNETWORK_SUBSCRIBERS_KEY` (along with `ACTIONNETWORK_KEY`).  You may also need to set `ACTIONNETWORK_SYNC=True`

File: docker-compose.yaml [hunk 2]
```diff
@@ -196,15 +197,15 @@ services:
...
-      - "9003:8000"
+  # docs:
+  #   build:
+  #     context: .
+  #     dockerfile: Dockerfile-docs
+  #   command: mkdocs serve -a 0.0.0.0:8000
+  #   volumes:
+  #     - ./docs:/docs
+  #   ports:
+  #     - "9003:8000"
```
Comment (liewegas) [inline docker-compose.yaml:208]: Did you mean to include the docker-compose changes?

Comment (benweissmann) [reply docker-compose.yaml:208]: Yes -- forgot to mention these changes in the PR description. I was having a lot of issues with my dev environment (very slow to reload), and these changes helped somewhat -- i just commented out non-critical containers (so you can un-comment them if you need them but they don't run by default) and combined all the workers into a single worker. Does that seem OK to you?

Comment (liewegas) [reply docker-compose.yaml:208]: Yep those changes look good!

File: app/subscription/forms.py [hunk 3]
```diff
@@ -1,18 +1,32 @@
...
+                enums.SubscriberPlan.FREE,
+                "Free: Embed VoteAmerica's tools on your own website and see aggregated stats, at no cost.",
+            ),
+            (
+                enums.SubscriberPlan.PREMIUM,
+                "Premium - $2,000/mo: Get detailed info on the voters who use VoteAmerica tools you embed on your website or link to, including email addresses and phone numbers to build your lists.",
+            ),
+            (
+                enums.SubscriberPlan.NONPROFIT,
+                "Nonprofit: 501(c)3 charitable organizations can use VoteAmerica's premium tools at a discount. Free until the end of 2020.",
```
Comment (liewegas) [inline app/subscription/forms.py:25]: Do we still want this 'free until end of 2020' bit?

**Review by liewegas — APPROVED** [review]
LGTM

File: app/subscription/models.py [hunk 4]
```diff
@@ -55,6 +56,10 @@ class Interest(
         null=True,
     )
 
+    # DEPRECATED
+    nonprofit = models.BooleanField(null=True)
+    product = models.ForeignKey(Product, null=True, on_delete=models.SET_NULL)
```
Comment (nickcatal) [inline app/subscription/models.py:61]: Let's make sure we remove these columns after this is deployed.

File: app/subscription/forms.py [hunk 5]
```diff
@@ -1,18 +1,32 @@
...
-        label="Term",
-        queryset=Product.objects.filter(public=True),
+    sms_mode = sms_mode_choice
+
+    plan = forms.ChoiceField(
+        choices=[
+            (
+                enums.SubscriberPlan.FREE,
+                "Free: Embed VoteAmerica's tools on your own website and see aggregated stats, at no cost.",
+            ),
```
Comment (nickcatal) [inline app/subscription/forms.py:18]: Can we remove this option on the interest form for right now? We need the subscription level available, but not sure that we want to explicitly offer it as an option when an account is setup right now.

**Review by nickcatal — APPROVED** [review]
Only flag is that we shouldn't offer an option for people to signup for a free tier account just yet. Otherwise, good!

**Pull request merged**